agent/engine/dependencygraph/graph.go

```diff
@@ -25,6 +25,14 @@ import (
 	"strings"
 )
 
+const (
+	ContainerStartDependencyCondition    = "START"
```

Comment:
we are already in dependency graph pacakges.. can we just call it StartCondition, etc

Reply:
makes sense. changed it.